airflow/contrib/hooks/aws_lambda_hook.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow.contrib.hooks.aws_hook import AwsHook
+
+
+class AwsLambdaHook(AwsHook):
+    """
+    Interact with AWS Lambda.
+    """
```

Comment:
You need some :paramname: and :type: documentation for the Class docstring, for the constructor properties
e.g.
:param function_name: The name of the AWS Lambda function to call
:type function_name: string

etc

Reply:
Good catch Thanks. ! I'll add docstrings today to both https://github.com/apache/incubator-airflow/pull/2570 and https://github.com/apache/incubator-airflow/pull/2586

Reply:
Looking through the contrib codebase I see a lot of missing docstrings  and param info, which is a shame because it's then missing when generating the docs, so maybe I will contribute to improving that.
 But generally I can't see any other issues with the code, so it's a great effort! At some point soon I will be using your dynamodb stuff heavily, so if it's not in a release by then I will borrow it for my plugins folder 😀

Reply:
thanks @davoscollective for all the positive energy :)